### ISSUE TITLE
[icerec] Incremental Release V05-01-07 (Trac #2055)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2047]
Bugfix for L2 offline processing: r157285

Migrated from https://code.icecube.wisc.edu/ticket/2055
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:44", 
    "description": "Bugfix for L2 offline processing: r157285", 
    "reporter": "joertlin", 
    "cc": "kjmeagher", 
    "resolution": "fixed", 
    "_ts": "1550067284370534", 
    "component": "combo reconstruction", 
    "summary": "[icerec] Incremental Release V05-01-07", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2017-07-27T15:52:26", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170727T16:05:07: joertlin commented:

> Jim Braun just brought up that we also should include a new gcdserver version that supports auth for mongodb.
> 
> That means that we need a new release of gcdserver (r157285) in this incremental release.

**Comment by icecube-trac** [comment]
Trac update at 20170731T11:55:25: kjmeagher commented:

> Can you make sure that the release notes for filterscripts and gcdserver are up-to-date?

**Comment by icecube-trac** [comment]
Trac update at 20170731T14:51:32: joertlin commented:

> Done

**Comment by icecube-trac** [comment]
Trac update at 20170803T15:02:36: 
* kjmeagher commented:

> In changeset 157507:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="157507"
> incremental release for icerec: includes updated gcdserver and filterscripts, closes #2047
> ```


* kjmeagher changed resolution from "" to "fixed"
* kjmeagher changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:44: 
* olivas changed milestone from "July 2017" to ""
* olivas commented:

> Milestone July 2017 deleted
